runFACTS.py

```diff
@@ -38,6 +37,11 @@ def run_experiment(exp_dir, debug_mode, resourcedir = None):
         # Exit
         sys.exit(0)
 
+    # Print out shell script if in shell script mode
+    if makeshellscript:
+        print_experimentsteps_script(experimentsteps)
```

Comment:
Running the `print_experimentsteps_script` function at this point will require the `onemodule` directory to include the `location.lst` file. Maybe there is a way to run this after line 86-90 or we need to be clear to the user that `makeshellscript = True` requires `location.lst` to be present in `exp_dir`

Reply:
Actually, even if I create a `location.lst` file in `experiments/onemodule/` the test run still fails with
```
FileNotFoundError: [Errno 2] No such file or directory: '/scratch/jt1187/test.1673016583/location.lst'
```
I think it is not being copied to the working directory.